tests/virtiofs/datavolume.go

```diff
@@ -335,5 +337,116 @@ var _ = Describe("[sig-storage] virtiofs", decorators.SigStorage, func() {
 			Expect(err).ToNot(HaveOccurred())
 			libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 		})
+
+		It("[Serial] should be successfully started and injected into the container", Serial, func() {
+			dataVolume := libdv.NewDataVolume(
+				libdv.WithRegistryURLSource(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine)),
+				libdv.WithStorage(
+					libdv.StorageWithStorageClass(sc),
+					libdv.StorageWithVolumeMode(k8sv1.PersistentVolumeFilesystem),
+				),
+				libdv.WithNamespace(testsuite.NamespaceTestDefault),
+			)
+
+			By("creating the DataVolume")
+			dataVolume, err = virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.NamespaceTestDefault).Create(context.Background(), dataVolume, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			By("waiting until the DataVolume is ready")
+			if libstorage.IsStorageClassBindingModeWaitForFirstConsumer(libstorage.Config.StorageRWOFileSystem) {
+				Eventually(ThisDV(dataVolume), 30).Should(WaitForFirstConsumer())
+			}
+
+			vmi = libvmifact.NewFedora(
+				libvmi.WithFilesystemDV(dataVolume.Name),
+				libvmi.WithNamespace(testsuite.NamespaceTestDefault),
+			)
+
+			vmi, err := kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			// with WFFC the run actually starts the import and then runs VM, so the timeout has to include both import and start
+			By("waiting until the DataVolume is ready")
+			libstorage.EventuallyDV(dataVolume, 500, HaveSucceeded())
+
+			By("waiting until the VirtualMachineInstance starts")
+			vmi = libwait.WaitForVMIPhase(vmi, []v1.VirtualMachineInstancePhase{v1.Running}, libwait.WithTimeout(500))
+
+			By("getting kubevirt client")
+			virtCli := kubevirt.Client()
+
+			By("creating a privileged pod")
+			podName := "info-pod"
+			debugPod := &k8sv1.Pod{
```

Comment:
I don't think you need this pod, you can use the virt-handler from the node where the VM is running

Reply:
ok, I will try